packages/commercetools/api-client/src/api/createMyShoppingList/index.ts

```diff
@@ -0,0 +1,36 @@
+
+import { MyShoppingListDraft } from './../../types/GraphQL';
+import { apolloClient, getSettings } from './../../index';
+import defaultQuery from './defaultMutation';
+import { getCustomQuery } from './../../helpers/queries';
+import gql from 'graphql-tag';
+
+interface ShoppingListData extends Omit<MyShoppingListDraft, 'currency'> {
+  currency?: string;
+}
+
+const createMyShoppingList = async (myShoppingListDraft: ShoppingListData, customQueryFn?) => {
```

Comment:
draft

Reply:
@andrzejewsky Can you remind me please what was the problem here?